qemu/tests/chardev_acpi.py

```diff
@@ -0,0 +1,36 @@
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    acpi description of serial and parallel ports incorrect
+    with -chardev/-device:
+    1) Boot guest A with isa-serial with tty chardev backend
+    2) Enter into guest A
+        udevadm info --query path --name /dev/ttyS0
+        !! --attribute-walk | grep looking
+    3) Boot guest B with -serial /dev/ttyS0
+    4) repeat step 2 for guest B
+    5) Check if the result are same for A and B.
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    output = []
+    timeout = float(params.get("login_timeout", 240))
+    p = 'udevadm info --query path --name /dev/ttyS0'
+    for x in range(2):
+        vm = env.get_vm(params["main_vm"])
+        if x >= 1:
+            params['serials'] = ''
+            params['extra_params'] = ' -serial /dev/ttyS0'
+            vm.create(params=params)
```

Comment:
For L23~L28, you want to create 2 vm, and the second vm with `-serial /dev/ttyS0`.
You can define vms = 'vm1 vm2' in the cfg.